assets/src/edit-story/components/panels/animation/directionRadioInput.js

```diff
@@ -0,0 +1,204 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { DIRECTION } from '../../../../animation';
+
+const Svg = styled.svg`
+  display: block;
+  height: 10px;
+  width: 10px;
+  fill: none;
+  transform-origin: 50% 50%;
+  transform: rotate(
+    ${({ direction }) => {
+      switch (direction) {
+        case DIRECTION.RIGHT_TO_LEFT:
+          return '270deg';
+        case DIRECTION.TOP_TO_BOTTOM:
+          return '180deg';
+        case DIRECTION.LEFT_TO_RIGHT:
+          return '90deg';
+        default:
+          return '0deg';
+      }
+    }}
+  );
+`;
+
+const Icon = styled.div`
+  padding: 4px;
+`;
+
+const Direction = ({ className, direction }) => (
+  <Icon className={className}>
+    <Svg viewBox="0 0 10 11" direction={direction}>
+      <path d="M5 11L5 1M5 1L9 5M5 1L1 5" />
+    </Svg>
+  </Icon>
+);
+
+Direction.propTypes = {
+  className: PropTypes.string,
+  direction: PropTypes.oneOf(Object.values(DIRECTION)),
+};
+
+// Must be a styled component to add component selectors in css
+const DirectionIndicator = styled(Direction)``;
+
+const Fieldset = styled.fieldset`
+  position: relative;
+  height: 63px;
+  width: 63px;
+  background-color: ${({ theme }) => theme.colors.bg.workspace};
+  border: 1px solid ${({ theme }) => theme.colors.fg.v9};
+  border-radius: 4px;
+`;
+
+const Figure = styled.div`
+  position: absolute;
+  display: block;
+  top: 50%;
+  left: 50%;
+  width: 18px;
+  height: 26px;
+  background: ${({ theme }) => theme.colors.fg.v9};
+  border-radius: 2px;
+  transform: translate(-50%, -50%);
+`;
+
+const Label = styled.label`
+  position: absolute;
+  cursor: pointer;
+
+  ${({ direction }) => {
+    switch (direction) {
+      case DIRECTION.RIGHT_TO_LEFT:
+        return css`
+          top: 50%;
+          right: 0;
+          transform: translateY(-50%);
+        `;
+      case DIRECTION.TOP_TO_BOTTOM:
+        return css`
+          top: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        `;
+      case DIRECTION.LEFT_TO_RIGHT:
+        return css`
+          top: 50%;
+          left: 0;
+          transform: translateY(-50%);
+        `;
+      default:
+        return css`
+          bottom: 0;
+          left: 50%;
+          transform: translateX(-50%);
+        `;
+    }
+  }}
+
+  ${DirectionIndicator} {
+    stroke: ${({ theme }) => theme.colors.fg.v9};
+    stroke-width: 1px;
+  }
+
+  input:checked ~ ${DirectionIndicator} {
+    stroke: ${({ theme }) => theme.colors.activeDirection};
+    stroke-width: 2px;
+  }
+
+  input:focus ~ ${DirectionIndicator} {
+    outline: 5px auto -webkit-focus-ring-color;
+  }
```

Comment:
This isn't showing an outline on firefox?

Reply:
Thanks for pointing out, will trouble shoot